### ISSUE TITLE
Controls do not work

### ISSUE THREAD

**Issue by uschmann** [dciabrin/ngdevkit#45]
I can not get any input to work in the examples.

I alreadyconfigured the following inmy *~/.gngeo* file:

```
p1control A=K122,B=K120,C=K97,D=K115,START=K49,COIN=K51,UP=K273,DOWN=K274,LEFT=K276,RIGHT=K275,MENU=K27
```

**Comment by dciabrin** [comment]
Which environment are you using? Linux? macOS?

**Comment by uschmann** [comment]
I am on ubuntu 20.04.1

**Comment by dciabrin** [comment]
The scan code that gngeo uses come from SDL2, their meaning is detailed in /usr/include/SDL2/SDL_scancode.h.
Based on your setting, your joystick would be accessible in weird keys:

```
    SDL_SCANCODE_AC_REFRESH = 273,
    SDL_SCANCODE_AC_BOOKMARKS = 274,
    SDL_SCANCODE_BRIGHTNESSDOWN = 275,
    SDL_SCANCODE_BRIGHTNESSUP = 276,
```

ngdevkit-examples comes with a user-friendly default config if you don't have a default ~/.gngeo/ngdevkit-gngeorc to load from:

```
p1control A=K97,B=K115,C=K113,D=K119,START=K49,COIN=K51,UP=K82,DOWN=K81,LEFT=K80,RIGHT=K79,A=J0B0,B=J0B1,C=J0B2,D=J0B3,START=J0B9,COIN=J0B8,UP=J0a3,DOWN=J0a3,LEFT=J0A0,RIGHT=J0A0
```

This one should work.

**Comment by uschmann** [comment]
For some reason the config is not loaded from **~/.gngeo/ngdevkit-gngeorc**
If i delete the .gngeo folder and start the emulator itś still not working.

I added the p1control flag to the gngeo command in Makefile.common

```
--p1control=A=K97,B=K115,C=K113,D=K119,START=K49,COIN=K51,UP=K82,DOWN=K81,LEFT=K80,RIGHT=K79,A=J0B0,B=J0B1,C=J0B2,D=J0B3,START=J0B9,COIN=J0B8,UP=J0a3,DOWN=J0a3,LEFT=J0A0,RIGHT=J0A0
```

This fixed it for me.

**Comment by dciabrin** [comment]
Sorry for the awfully late answer...

I just realized that the nightly debian packages were built incorrectly so the binaries were defaulting to read `~/.gngeo/gngeorc`, rather than the ngdevkit-specific file `~/.gngeo/ngdevkit-gngeorc`.

I have fix the debian rules, the fix is available starting ngdevkit-gngeo 0.8.1+202105061001-10 for Focal, Hirsute and Impish.
